src/localization.ts

```diff
@@ -14,19 +14,25 @@ distributed under the License is distributed on an "AS IS" BASIS,
 limitations under the License.
 */
 
-const sourceMap = require('source-map');
-const fs = require('fs');
+import {SourceMapConsumer} from 'source-map';
+import * as fs from 'fs';
+
+interface Location {
+    path: string;
+    line: number;
+    column: number;
+}
 
 /**
  * Find the path of a source TypeScript file on the disk
  * for a JS file in the web app.
  *
- * @param {string} location
+ * @param {Array<string>} location
```

Comment:
We usually drop the parameter annotations in jsdoc for TS code (or at least the _type_ of the arguments), so you could either leave only the description of the method, or set `@param name description` for some params whose meaning is not obvious, when looking at their name.